src/main/java/net/rptools/maptool/model/CampaignProperties.java

```diff
@@ -163,6 +161,11 @@ public void mergeInto(CampaignProperties properties) {
     properties.tokenStates.putAll(tokenStates);
     properties.tokenBars.putAll(tokenBars);
     properties.defaultTokenPropertyType = defaultTokenPropertyType;
+    if (properties.gridFootprints != null) {
+      properties.gridFootprints.putAll(gridFootprints);
+    } else {
+      properties.gridFootprints = gridFootprints;
+    }
```

Comment:
Should create a new HashMap instead of assigning so that if its modified after the call it wont cause issues.